api/versionedvalidatorregistration.go

```diff
@@ -24,7 +25,7 @@ import (
 
 // VersionedValidatorRegistration contains a versioned ValidatorRegistrationV1.
 type VersionedValidatorRegistration struct {
-	Version   spec.DataVersion
+	Version   spec.BuilderVersion
 	Bellatrix *apiv1.ValidatorRegistration
```

Comment:
This field should map to the builder version, so be called `V1` rather than `Bellatrix`.